<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>800</width>
    <height>600</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <layout class="QGridLayout" name="gridLayout">
    <item row="0" column="0" rowspan="3" colspan="2">
     <widget class="QCustomPlot" name="widget" native="true">
      <property name="maximumSize">
       <size>
        <width>16777215</width>
        <height>16777215</height>
       </size>
      </property>
      <property name="baseSize">
       <size>
        <width>0</width>
        <height>0</height>
       </size>
      </property>
      <property name="font">
       <font>
        <hintingpreference>PreferDefaultHinting</hintingpreference>
        <fontweight>Medium</fontweight>
       </font>
      </property>
     </widget>
    </item>
    <item row="8" column="1">
     <layout class="QVBoxLayout" name="verticalLayout">
      <property name="spacing">
       <number>6</number>
      </property>
      <property name="sizeConstraint">
       <enum>QLayout::SetDefaultConstraint</enum>
      </property>
      <item>
       <widget class="QLabel" name="label">
        <property name="sizePolicy">
         <sizepolicy hsizetype="Preferred" vsizetype="Fixed">
          <horstretch>0</horstretch>
          <verstretch>0</verstretch>
         </sizepolicy>
        </property>
        <property name="font">
         <font>
          <pointsize>12</pointsize>
          <bold>true</bold>
         </font>
        </property>
        <property name="text">
         <string>Размер хеш-таблиц</string>
        </property>
        <property name="alignment">
         <set>Qt::AlignCenter</set>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QCheckBox" name="checkBox16">
        <property name="font">
         <font>
          <pointsize>12</pointsize>
          <bold>true</bold>
         </font>
        </property>
        <property name="text">
         <string>16</string>
        </property>
        <property name="checked">
         <bool>true</bool>
        </property>
        <attribute name="buttonGroup">
         <string notr="true">buttonGroup</string>
        </attribute>
       </widget>
      </item>
      <item>
       <widget class="QCheckBox" name="checkBox64">
        <property name="font">
         <font>
          <pointsize>12</pointsize>
          <bold>true</bold>
         </font>
        </property>
        <property name="text">
         <string>64</string>
        </property>
        <attribute name="buttonGroup">
         <string notr="true">buttonGroup</string>
        </attribute>
       </widget>
      </item>
      <item>
       <widget class="QCheckBox" name="checkBox128">
        <property name="font">
         <font>
          <pointsize>12</pointsize>
          <bold>true</bold>
         </font>
        </property>
        <property name="text">
         <string>128</string>
        </property>
        <attribute name="buttonGroup">
         <string notr="true">buttonGroup</string>
        </attribute>
       </widget>
      </item>
      <item>
       <widget class="QCheckBox" name="checkBox2048">
        <property name="font">
         <font>
          <pointsize>12</pointsize>
          <bold>true</bold>
         </font>
        </property>
        <property name="text">
         <string>2048</string>
        </property>
        <attribute name="buttonGroup">
         <string notr="true">buttonGroup</string>
        </attribute>
       </widget>
      </item>
      <item>
       <layout class="QHBoxLayout" name="horizontalLayout">
        <item>
         <widget class="QLabel" name="label_2">
          <property name="font">
           <font>
            <pointsize>14</pointsize>
            <bold>true</bold>
           </font>
          </property>
          <property name="text">
           <string>Количество замеров</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QLCDNumber" name="amountNumber">
          <property name="sizePolicy">
           <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
            <horstretch>0</horstretch>
            <verstretch>0</verstretch>
           </sizepolicy>
          </property>
          <property name="autoFillBackground">
           <bool>false</bool>
          </property>
          <property name="styleSheet">
           <string notr="true">color: rgb(0, 0, 0);</string>
          </property>
          <property name="frameShape">
           <enum>QFrame::Box</enum>
          </property>
          <property name="frameShadow">
           <enum>QFrame::Plain</enum>
          </property>
          <property name="lineWidth">
           <number>1</number>
          </property>
          <property name="midLineWidth">
           <number>0</number>
          </property>
          <property name="smallDecimalPoint">
           <bool>false</bool>
          </property>
          <property name="digitCount">
           <number>10</number>
          </property>
          <property name="segmentStyle">
           <enum>QLCDNumber::Flat</enum>
          </property>
         </widget>
        </item>
       </layout>
      </item>
     </layout>
    </item>
    <item row="8" column="0">
     <layout class="QVBoxLayout" name="verticalLayout_3">
      <item>
       <layout class="QVBoxLayout" name="verticalLayout_2">
        <item>
         <widget class="QLabel" name="label_3">
          <property name="font">
           <font>
            <pointsize>12</pointsize>
            <bold>true</bold>
           </font>
          </property>
          <property name="text">
           <string>Количество колизий у HashModTable и HashMultTable</string>
          </property>
         </widget>
        </item>
        <item>
         <layout class="QHBoxLayout" name="horizontalLayout_3">
          <item>
           <widget class="QLCDNumber" name="amountModCollisions">
            <property name="styleSheet">
             <string notr="true">color: rgb(0, 0, 0);</string>
            </property>
            <property name="frameShadow">
             <enum>QFrame::Plain</enum>
            </property>
            <property name="digitCount">
             <number>10</number>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QLCDNumber" name="amountMultCollisions">
            <property name="styleSheet">
             <string notr="true">color: rgb(0, 0, 0);</string>
            </property>
            <property name="frameShadow">
             <enum>QFrame::Plain</enum>
            </property>
            <property name="digitCount">
             <number>10</number>
            </property>
           </widget>
          </item>
         </layout>
        </item>
       </layout>
      </item>
      <item>
       <widget class="QPushButton" name="clearButton">
        <property name="font">
         <font>
          <pointsize>14</pointsize>
          <bold>true</bold>
         </font>
        </property>
        <property name="cursor">
         <cursorShape>PointingHandCursor</cursorShape>
        </property>
        <property name="text">
         <string>Очистить замеры</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QPushButton" name="buildButton">
        <property name="font">
         <font>
          <pointsize>14</pointsize>
          <bold>true</bold>
         </font>
        </property>
        <property name="cursor">
         <cursorShape>PointingHandCursor</cursorShape>
        </property>
        <property name="text">
         <string>Постоить график</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QPushButton" name="timeFindButton">
        <property name="font">
         <font>
          <pointsize>14</pointsize>
          <bold>true</bold>
         </font>
        </property>
        <property name="cursor">
         <cursorShape>PointingHandCursor</cursorShape>
        </property>
        <property name="text">
         <string>Произвести замер поиска у классов</string>
        </property>
       </widget>
      </item>
     </layout>
    </item>
   </layout>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>800</width>
     <height>22</height>
    </rect>
   </property>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <customwidgets>
  <customwidget>
   <class>QCustomPlot</class>
   <extends>QWidget</extends>
   <header>qcustomplot.h</header>
   <container>1</container>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections/>
 <buttongroups>
  <buttongroup name="buttonGroup"/>
 </buttongroups>
</ui>
